paasta_tools/monitoring/check_mesos_outdated_tasks.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+# Copyright 2015-2017 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import datetime
+import sys
+import time
+
+from paasta_tools.mesos_tools import get_mesos_master
+from paasta_tools.utils import load_system_paasta_config
+
+
+OUTPUT_FORMAT = "{:<30}  {:<8}  {:<20}  {:<27}  {}"
+
+
+def get_mesos_state():
+    state = get_mesos_master().state
```

Comment:
:( can we used the caching stuff that maksym wrote?

Further, can we just call the /frameworks endpoint, rather than getting the whole state?

Reply:
@keymone could you explain please how does the caching works?
I tried to pass use_mesos_cache=True to get_mesos_master, but it failed because nothing was listening on 5055.
https://github.com/Yelp/paasta/blob/master/paasta_tools/mesos/master.py#L76

Reply:
@oktopuz yeah, it's not deployed everywhere. passing `use_mesos_cache=True` is a way to opt-in if you know that caching is deployed, ideally we'd have that config option in mesos configs. also /slaves endpoint is not being cached.

Reply:
@Rob-Johnson I'll configure this check to use caching endpoint when we have them everywhere.
For now I don't expect us to run this check more often than once an hour. That should be safe for masters.

Reply:
I replaced it with
```
state = get_mesos_master(use_mesos_cache=True).state
```